src/engine.rs

```diff
@@ -183,7 +183,14 @@ pub fn run_treefmt(
 
                 match formatter.clone().fmt(&paths) {
                     // FIXME: do we care about the output?
-                    Ok(_) => {
+                    Ok(out) => {
```

Comment:
it's probably best to use `out.status.success()` since it's available on all platforms. Then special-case the error message whenever status.code() exists or not.